worker/pkg/benthos/transformers/generator_utils.go

```diff
@@ -139,24 +143,44 @@ func ParseBloblangSpec(benthosSpec *BenthosSpec) (*ParsedBenthosSpec, error) {
 					}
 				}
 				param := &BenthosSpecParam{
-					TypeStr:     lowercaseFirst(matches[1]),
-					Name:        toCamelCase(matches[2]),
-					IsOptional:  strings.Contains(line, ".Optional()"),
-					HasDefault:  defaultVal != "",
-					Default:     defaultVal,
-					Description: description,
+					TypeStr:      lowercaseFirst(matches[1]),
+					Name:         toCamelCase(matches[2]),
+					BloblangName: matches[2],
+					IsOptional:   strings.Contains(line, ".Optional()"),
+					HasDefault:   defaultVal != "",
+					Default:      defaultVal,
+					Description:  description,
 				}
 				params = append(params, param)
 			}
 		}
 	}
 
+	bloblangFuncName, err := extractBloblangFunctionName(benthosSpecStr)
+	if err != nil {
+		return nil, err
+	}
+
 	return &ParsedBenthosSpec{
-		Params:          params,
-		SpecDescription: specDescription,
+		BloblangFuncName: bloblangFuncName,
+		Params:           params,
+		SpecDescription:  specDescription,
 	}, nil
 }
 
+func extractBloblangFunctionName(input string) (string, error) {
+	// Looks for bloblang.RegisterFunctionV2 and captures the function name in quotes
+	re := regexp.MustCompile(`bloblang\.RegisterFunctionV2\("([^"]+)"`)
+
+	matches := re.FindStringSubmatch(input)
+
+	if len(matches) > 1 {
+		return matches[1], nil
+	}
+
+	return "", fmt.Errorf("bloblang function name not found")
```

Comment:
Is there a way this can say what the function name is? Is that the input? just curious if it would make this easier to understand ifwe knew what functionname was failed to be found.